public/js/sensorsChart.js

```diff
@@ -0,0 +1,67 @@
+const sensorsChart = document.getElementById("sensorsChart").getContext("2d");
```

Comment:
Which chart is this?

Reply:
@kilemensi Um, Is the sensors coverage chart. I'll probably rename it  to match its  equivalent js files

Reply:
There is already sensors coverage chart in `public/js/charts/sensorsCoverage.js` @nerdlet

Reply:
@kilemensi Ahhh, okay...deleted it